### ISSUE TITLE
Recovering session or viewing history

### ISSUE THREAD

**Issue by mhlr** [arunkumar9t2/chromer#33]
I use Chromer in web heads mode. Occasionally I have had all webheads disappear, prseumably due to a crash. Some I think sometimes it Chromer has asked me to reopen old tabs when restarted but not today.
Is there a way to try and make it reopen recently closed tabs or at least view history? I can not find anything like that in the menus

**Comment by arunkumar9t2** [comment]
Yes, currently there is no good means to recover unopened web heads. I will consider this on priority for next feature update which will be 1.8.

**Comment by mhlr** [comment]
please make this a priority. I have had a number of crashes where I have lost a significant number unopened webheads. My usual workflow is to open links from various feeds in other apps and then process  them in chromer. If I lose a whole session that wastes all the work I did before that.

**Comment by OriR** [comment]
Hey @arunkumar9t2 
Love this browser!
But that's a major issue I'm facing, like a desktop browser, I tend to horde my tabs to read them later. Because I can't restore the tabs, it means I have to "live in fear" that my phone will might shut down and I'll lose all of my open tabs.

Any chance we can make this happen?

**Comment by dragon788** [comment]
I've found that if the tabs are still open according to the notification but the bubble has disappeared, often if you open Lynket it shows any active/opened tabs there, but you can also rotate the device to landscape and then back to portrait and that sometimes gets the bubble stack to reappear.
